components/rpill/browser/rpill.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <locale>
+#include <string>
```

Comment:
this should go after the file header https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes